stories/RefreshCache.stories.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { InstantSearch, SearchBox } from '../packages/react-instantsearch/dom';
+import { CustomHits } from './util';
+
+const stories = storiesOf('RefreshCache', module);
```

Comment:
is that normal syntax? What is `module`?

Reply:
Yes, normal syntax, see https://storybook.js.org/basics/guide-react/#create-the-config-file